src/server/game/Entities/Player/Player.h

```diff
@@ -1115,6 +1115,15 @@ class Player : public Unit, public GridObject<Player>
         SetFloatValue(UNIT_FIELD_BOUNDINGRADIUS, scale * DEFAULT_WORLD_OBJECT_SIZE);
         SetFloatValue(UNIT_FIELD_COMBATREACH, scale * DEFAULT_COMBAT_REACH);
     }
+	
+	// Detect Client Languaje
+    bool ClientInSpanishOrMexican()
+    {
+        if (GetSession()->GetSessionDbLocaleIndex() == LOCALE_esES || GetSession()->GetSessionDbLocaleIndex() == LOCALE_esMX)
+            return true;
+        else
+            return false;
+    }
```

Comment:
where is this method supposed to be used?

Reply:
@FrancescoBorzi the idea is to use it for the greater scope of the modules.

Reply:
Personally, it has been quite useful to be able to offer menus in both Spanish and English

Reply:
For example in the TemplateNPC module.
```
           if (sTemplateNpcMgr->CanEquipTemplate(player, playerSpecStr) == false)
            {
                if (player->ClientInSpanishOrMexican())
                {
                    player->GetSession()->SendAreaTriggerMessage("No hay plantillas para esta especializacion %s .", playerSpecStr.c_str());
                }
                else
                {
                    player->GetSession()->SendAreaTriggerMessage("There's no templates for %s specialization yet.", playerSpecStr.c_str());
                }
                
                return;
```